services/venice-controller/src/test/java/com/linkedin/venice/controller/server/TestAdminSparkServer.java

```diff
@@ -997,10 +997,13 @@ public void testCleanupInstanceCustomizedStates() {
     String storeName = Utils.getUniqueString("cleanupInstanceCustomizedStatesTest");
     VeniceHelixAdmin childControllerAdmin = cluster.getRandmonVeniceController().getVeniceHelixAdmin();
     childControllerAdmin.createStore(clusterName, storeName, "test", "\"string\"", "\"string\"");
-    childControllerAdmin.incrementVersionIdempotent(clusterName, storeName, "test", 1, 1);
+    Version version = childControllerAdmin.incrementVersionIdempotent(clusterName, storeName, "test", 1, 1);
     MultiStoreTopicsResponse response = controllerClient.cleanupInstanceCustomizedStates();
     Assert.assertFalse(response.isError());
-    Assert.assertTrue(response.getTopics().isEmpty());
+    Assert.assertNotNull(response.getTopics());
+    for (String topic: response.getTopics()) {
```

Comment:
It is not completely correct to check for a substring. For tests it is probably okay, but this is not a mirror of how things are in practice.

Can we instead be a bit stricter? This should work IMO - 
```
Assert.assertNotEquals(Version.parseStoreFromKafkaTopicName(topic), store)
```

Reply:
`Admin::cleanupInstanceCustomizedStates` returns list in which each element has the following form:
```
/venice-cluster_1eecc278875de_f024464/INSTANCES/sumane-mn1.linkedin.biz_58715/CUSTOMIZEDSTATES/OFFLINE_PUSH/<store-version>

# Example

/venice-cluster_1eecc278875de_f024464/INSTANCES/sumane-mn1.linkedin.biz_58715/CUSTOMIZEDSTATES/OFFLINE_PUSH/cleanupInstanceCustomizedStatesTest_1eecf18a1c477_de49a66_v1
```

Since in this tests we want to verify that the relevant store version's CS is not cleaned up, I think 
`Assert.assertFalse(topic.contains(version.Kafka topic name()));` is stricter than the `Assert.assertNotEquals(Version.parseStoreFromKafkaTopicName(topic), store)` as the later relies on store name rather than the store version.

Reply:
Sure. Let's try to be as strict as we can - without getting affected by other tests